modules/transactionpool/persist.go

```diff
@@ -53,6 +55,9 @@ func (tp *TransactionPool) initPersist() error {
 	}
 
 	// Open the database file.
+	if build.DEBUG && build.Release != "testing" {
+		fmt.Println("\tTransaction Pool: opening database")
+	}
```

Comment:
It only prints if you are running the debugging build, and I think it's useful to have this information coming from anyone running a `siad` instance - if it's having trouble starting up you want to know more about why.

It's not an overwhelming amount of information, and the bug is rare so we have a better chance of catching it if the code is in more places.